src/test/java/PinGuesserTest.kt

```diff
@@ -38,7 +35,7 @@ internal class PinGuesserTest {
             return java.util.Set.of(*input.split(", ".toRegex()).toTypedArray())
         }
 
-        private fun testSingleDigitParameters(): Stream<Arguments> {
+        @JvmStatic fun testSingleDigitParameters(): Stream<Arguments> {
```

Comment:
<img width="1526" alt="from-java-to-kotlin_–_PinGuesserTest_kt" src="https://user-images.githubusercontent.com/459464/118160523-b3977c00-b41e-11eb-8151-a4dea10aa9e2.png">